src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2950,6 +2950,26 @@ private void AddSynthesizedRecordMembersIfNecessary(MembersAndInitializersBuilde
                     members.Add(property.BackingField);
                 }
             }
+
+            var objEquals = new SynthesizedRecordObjEquals(this);
```

Comment:
Please consider breaking up these different members into local functions for readability.